ui-admin/src/study/export/ExportDataBrowser.tsx

```diff
@@ -108,7 +106,25 @@ const ExportDataBrowser = ({ studyEnvContext }: {studyEnvContext: StudyEnvContex
     </div>
     <ExportDataModal studyEnvContext={studyEnvContext} show={showExportModal} setShow={setShowExportModal}/>
     <LoadingSpinner isLoading={isLoading}/>
-    {!isLoading && basicTableLayout(table)}
+    {!data && <div className={'d-flex justify-content-center'}>
+      <Button
+        variant="secondary"
+        onClick={loadPreview}
+      >
+        Click to load data export preview...
```

Comment:
UX might be cleaner if this button was just called "Show preview" and had `outline={true}` so it looked a bit more buttony